data/json/mutations/mutations.json

```diff
@@ -886,19 +886,30 @@
   {
     "type": "mutation",
     "id": "ADRENALINE2",
-    "name": { "str": "Combat Adaptation" },
+    "name": { "str": "Berserk" },
     "points": 3,
-    "description": "When you are in great danger you may flip a chemical switch in your brain and for a cost in hunger, fatigue, and thirst your nervous system will process things around you slightly faster.  Giving you better reaction times that you can pay for later if you survive.",
+    "description": "When you are in great danger, you may flip a chemical switch in your brain and for a cost in hunger, fatigue, and thirst your strength will be greatly increased for a random time between 2 and 5 minutes.  However, your mind will become so clouded that dodging will become impossible, and the process will take a toll on your body.  The crash will last 10 minutes.",
     "prereqs": [ "ADRENALINE" ],
     "category": [ "BEAST" ],
     "active": true,
     "cost": 90,
-    "time": "10 s",
     "fatigue": true,
     "kcal": true,
     "thirst": true,
-    "dodge_modifier": 2,
-    "movecost_modifier": 0.9
+    "transform": { "target": "ADRENALINE2_active", "msg_transform": "You go berserk!", "active": true }
+  },
+  {
+    "type": "mutation",
+    "id": "ADRENALINE2_active",
+    "name": { "str": "Berserk (active)" },
+    "points": 3,
+    "description": "A red film covers your vision, and your mind is empty except for one thought: Kill.  Kill.  Kill.  Kill.  Kill.  Kill.  Kill.",
+    "prereqs": [ "ADRENALINE" ],
+    "category": [ "BEAST" ],
+    "dodge_modifier": -20,
+    "movecost_modifier": 0.7,
```

Comment:
This isn't +30% speed, this is -30% walking cost. Check "Quick" for the format to add Speed.

Reply:
...actually I like this more, it fits the flavor to be able to get past obstacles much faster, more _Speed_ speed would just make the buffs even stronger.